bftengine/src/communication/TlsTCPCommunication.cpp

```diff
@@ -1138,7 +1151,6 @@ int TlsTCPCommunication::Stop() {
     return 0;
 
   auto res = _ptrImpl->Stop();
```

Comment:
nit: `return _ptrImpl->Stop();` rather than using a temporary variable.

Reply:
I found that it is much easier to debug when you have temp variable.